sale_order_merge/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Merge sale orders
+=================
+
+This module allows the sale employee to merge draft or confirmed orders
+from the same customer.
+
+When orders are merged, draft invoices and unprocessed outgoing pickings
+will be merged as well.
+
+Usage
+=====
+
+To use this module, you need to go to the main sale order that you want to
+merge with other sale orders. If there are any candidates, you will see a
+*Merge* button in the header of the sale order form. If you click on this
+button, a window will pop-up that has all mergeable sale orders preselected.
+Remove the orders that you do not want to merge, and click on the *Merge*
+button in the footer of the pop-up window. The main window will then refresh
+to the updated main sale order.
+
```

Comment:
I suggest to add the following:

The mergeability criteria is defined as follows:
- Same customer, shipping address, warehouse and company.
- Orders in status 'Draft Quotation', 'Quotation Sent', 'Waiting Shedule', 'Sale Order', 'Sale to Invoice'.
- Once the order has already been confirmed, only draft orders or confirmed orders with the same invoice policy can be merged.

The criteria can easily be extended using method `_get_merge_domain` of model `sale.order`